src/services/item/plugins/embeddedLink/schemas.ts

```diff
@@ -1,8 +1,10 @@
-import S from 'fluent-json-schema';
+import Ss from 'fluent-json-schema';
 
 import { ItemType } from '@graasp/sdk';
 
-import { error } from '../../../../schemas/fluent-schema';
+import { error } from '../../../../schemas/fluent-schema.js';
+
+const S = Ss.default;
```

Comment:
Same comment as for `waitFor`.

Reply:
This one was also fixed by using the named import instead of the default.